.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -167,6 +167,9 @@ jobs:
             VERSION: ${{ github.event.release.name }}
 
         steps:
+            - name: Install Subversion
+              run: |
+                  apt-get update -y && apt-get install -y subversion
```

Comment:
This is probably good assuming `svn` doesn't get added back to these boxes, otherwise it will be a little wasteful.

Are there any circumstances where the `upload-tag` job is run below without first running the `upload` job? If so, this should be handled in it's own job.

Reply:
Ah, I see. Both run in `github.event.release.assets[0]` is truthy, but which one depends on whether `needs.compute-should-update-trunk.outputs.should_update_trunk` is `true` or not. 

Perhaps it's better to add a job prior to these two that run `if: github.event.release.assets[0]` assuming both jobs run on the same environment. @desrosj what would you recommend?

Reply:
> Perhaps it's better to add a job prior to these two that run `if: github.event.release.assets[0]` assuming both jobs run on the same environment. @desrosj what would you recommend?

Do you mean have a separate job before `upload` and `upload-tag` that installs SVN? If so, that unfortunately does not work. Each job is run on a separate, fresh runner machine. So we would need to add the step above to both of those jobs.

Reply:
Updated in [04089c5](https://github.com/WordPress/gutenberg/pull/68837/commits/04089c5aef4bfdf7acdd9b440b45493ebcea38c1).